modules/ROOT/pages/6.0-release-notes-premium-changes.adoc

```diff
@@ -3,16 +3,176 @@
+
+| `images_upload_timeout`             | `editimage_upload_timeout`       | Option                              | NB: _Image Tools_ is now _Enhanced Image Editing_, a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].
+
+| `toc`                               | `tableofcontents`                | Plugin, Menu item, and Toolbar item | This presents in both the menu item and the toolbar’s tooltip text. NB: _Table of Contents_ is now a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].
+
+| `tocupdate`                         | `tableofcontentsupdate`          | Toolbar item                        | This presents in the toolbar’s tooltip text. NB: _Table of Contents_ is now a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].
+
+| `toc_class`                         | `tableofcontents_class`          | Option                              | NB: _Table of Contents_ is now a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].
+
+| `toc_depth`                         | `tableofcontents_depth`          | Option                              | NB: _Table of Contents_ is now a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].
+
+| `toc_header`                        | `tableofcontents_header`         | Option                              | NB: _Table of Contents_ is now a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].
+|===
+
+
+
 // tag::accompanying-premium-plugin-changes[]
 [[accompanying-premium-plugin-changes]]
 == Accompanying Premium Plugin changes
 
 The following premium plugin updates were released alongside {productname} 6.0.
 
+=== Enhanced Image Editing (was Image Tools)
+
+As noted in the xref:6.0-release-notes-core-changes.adoc#removed-or-deprecated-plugins-imagetools[_{productname} 6.0 Release Notes_], the `imagetools` plugin is no longer included as part of the Free editor.
+
+_Image Tools_ is now _Enhanced Image Editing_, a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].
+
+The `imagetools_proxy` option has been renamed to `editimage_proxy`. However, when setting up xref:introduction-to-premium-selfhosted-services.adoc[Tiny services] the recommended configuration is now `editimage_proxy_service_url` and will be used in favour of `editimage_proxy` if both are set.
+
+Users of the Free version of {productname} 5.x who rely on the removed `imagetools` plugin should not upgrade to {productname} 6.0 without a https://tiny.cloud/pricing/[subscription plan] that provides access to https://tiny.cloud/tinymce/features/#productivity[_Enhanced Image Editing_].
+
+=== Spell Checker Pro
+
+The free `spellchecker` plugin was removed as no longer supported.
+
+The https://tiny.cloud/tinymce/features/#productivity[Premium plugin], xref:introduction-to-tiny-spellchecker.adoc[Spell Checker Pro], which offers equivalent functionality and more, is available.
```

Comment:
This is more talking about the old open source spellchecker which is a little confusing but we can fix that later.